cmd/aries-agent-mobile/pkg/wrappers/storage/storage.go

```diff
@@ -32,6 +32,17 @@ func (p *Provider) OpenStore(name string) (storage.Store, error) {
 
 type store struct{ api.Store }
 
+// Get fetches the record based on key.
+func (s *store) Get(k string) ([]byte, error) {
+	res, err := s.Store.Get(k)
+
+	if err != nil && err.Error() == storage.ErrDataNotFound.Error() {
```

Comment:
Can we use `errors.Is(err, storage.ErrDataNotFound)` here ?

Reply:
It will not work here. We need to check strings here.